Main-Page

teorge opened the pull request:
Completed main page of UberEats layout

evgenyrodionov commented:
утром проверю

evgenyrodionov commented:
https://github.com/teorge/ubereats-demo-frontend/pull/2#issuecomment-394185337

evgenyrodionov commented on index.html:
```diff
@@ -0,0 +1,372 @@
+      <div class="row">
+        <div class="col-xs-12">
+
+        </div>
```
это что?

evgenyrodionov commented on index.html:
```diff
@@ -0,0 +1,372 @@
+              </select>
+            </div>
+
+            <div class="col-xs-9">
```
почему колонки используются не как дети .row?

evgenyrodionov commented on index.html:
```diff
@@ -0,0 +1,372 @@
+          </form>
+        </div>
+
+        <div class="col-xs-12 col-lg-12">
```
![image](https://user-images.githubusercontent.com/775692/40994115-15d5b788-6904-11e8-924a-2c2abb2e0a6b.png)

evgenyrodionov commented on src/css/style.css:
```diff
@@ -0,0 +1,709 @@
+  min-width: 302px;
+}
+
+/* Flexbox grid */
```
![image](https://user-images.githubusercontent.com/775692/40994142-2daba282-6904-11e8-923c-536f130cac31.png)

evgenyrodionov commented on src/css/style.css:
```diff
@@ -0,0 +1,709 @@
+}
+
+.location__urgency {
+  width: 65px;
```
только для изображений

evgenyrodionov commented on package.json:
```diff
@@ -0,0 +1,15 @@
+  "license": "MIT",
+  "devDependencies": {
+    "autoprefixer": "^8.4.1",
+    "gulp": "^3.9.1",
```
а где галпфайл

teorge commented:
What was done:
 - Migration to flexboxgrid2;
 - Removed unnecessary grid elements and comments
 - Replaced fixed spacing with paddings (box model), also changed unicode symbols from HTML-code to their graphic representation
 - Fixed horizontal rules to hr tag and changed positioning in some elements
 - Added gulpfile.js and gulp-watch

evgenyrodionov commented:
теперь осталось определиться как мерджить работу
